game/scripts/npc/items/item_havoc_hammer.txt

```diff
@@ -0,0 +1,117 @@
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilityCooldown"                                     "10.0"
+
+    // Item Info
+    //-------------------------------------------------------------------------------------------------------------
+    "ItemCost"                                            "15550" //OAA
+    "ItemIsNeutralDrop"                                   "0" //OAA
+    "ItemPurchasable"                                     "1" //OAA
+    "ItemShopTags"                                        "str;damage"
+    "ItemQuality"                                         "artifact"
+    "ItemDisassembleRule"                                 "DOTA_ITEM_DISASSEMBLE_NEVER"
+    "ItemAliases"                                         "havoc hammer"
+    "ItemDeclarations"                                    "DECLARE_PURCHASES_TO_TEAMMATES | DECLARE_PURCHASES_IN_SPEECH | DECLARE_PURCHASES_TO_SPECTATORS"
+    "ItemShareability"                                    "ITEM_NOT_SHAREABLE"
+    "MaxUpgradeLevel"                                     "2"
+    "ItemBaseLevel"                                       "1"
+
+    "Model"                                               "models/props_gameplay/neutral_box.vmdl"
+
+    // Special
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilitySpecial"
+    {
+      "01"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "bonus_damage"                                    "65 105" //OAA
+      }
+      "02"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "bonus_strength"                                  "80 120" //OAA
+      }
+      "03"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "range"                                           "350"
+      }
+      "04"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "slow"                                            "50"
+      }
+      "05"
+      {
+        "var_type"                                        "FIELD_FLOAT"
+        "slow_duration"                                   "3"
+      }
+      "06"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "angle"                                           "360"
+      }
+      "07"
+      {
+        "var_type"                                        "FIELD_FLOAT"
+        "knockback_duration"                              "0.3"
+      }
+      "08"
+      {
+        "var_type"                                        "FIELD_FLOAT"
+        "knockback_distance"                              "250"
+      }
+      "09"
+      {
+        "var_type"                                        "FIELD_INTEGER"
+        "nuke_base_dmg"                                   "225 375"
```

Comment:
This base damage is very low. I would increase the base damage to "400 600". The base damage shouldnt be the same as its dota counterpart. I dont think the STR scaling bonus damage will be very significant.

Reply:
I want to see it in game first before making theories if its good or not.